js/src/pages/price-benchmark/empty-metrics-notice/index.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import { EmptyTable } from '@woocommerce/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppDocumentationLink from '~/components/app-documentation-link';
+
+/**
+ * Internal dependencies
+ */
+import EmptyMetricsGraphic from '~/images/price-benchmark/empty-metrics.svg';
+
+const EmptyMetricsNotice = () => {
+	return (
+		<EmptyTable>
+			<div className="gla-price-benchmark__empty-metrics">
```

Comment:
@ankitrox I'd rather have `EmptyTable`  within the `gla-price-benchmark__empty-metrics` wrapper.

Reply:
That way we the things won't be centere aligned anymore and will need more of CSS. We can keep it as is for now in case it is not breaking anything or impacting any other area.

Reply:
@ankitrox True, however it's weird having the `EmptyMetricsNotice` component which when you inspect the markup, renders the markup for `EmptyTable`. I feel it's incorrect. Here is what I suggest:

- Have the wrapper container `.gla-price-benchmark__empty-metrics` wrap `EmptyTable`.
- Within `EmptyTable` define a new HTML container which has the same CSS as `.gla-price-benchmark__empty-metrics`. The CSS for `.gla-price-benchmark__empty-metrics` should be removed.
- The content will still be centered.

The overall layout will be like this:
```js
.gla-price-benchmark__empty-metrics
	EmptyTable
		.gla-price-benchmark__empty-metrics-columns
			.gla-price-benchmark__empty-metrics-graphic
			.gla-price-benchmark__empty-metrics-description
```